CommunityToolShedMvc/CommunityToolShedMvc/Controllers/AccountController.cs

```diff
@@ -0,0 +1,93 @@
+﻿using CommunityToolShedMvc.Data;
+using CommunityToolShedMvc.Models;
+using CommunityToolShedMvc.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using System.Web.Security;
+
+namespace CommunityToolShedMvc.Controllers
+{
+    [Authorize]
+    public class AccountController : Controller
+    {
+        [AllowAnonymous]
+        public ActionResult Login()
+        {
+            
+            var viewModel = new LoginViewModel();
+            return View(viewModel);
+        }
+
+        [AllowAnonymous]
+        public ActionResult Register()
+        {
+            var viewModel = new RegisterViewModel();
+            return View(viewModel);
+        }
```

Comment:
It's a convention to keep the GET and POST versions of action methods next to each other (i.e. the GET `Login` action method is followed by the POST `Login` method). Doing this makes it easier to read and follow your code.